native_client/swift/deepspeech_ios/deepspeech_ios.modulemap

```diff
@@ -0,0 +1,12 @@
+framework module deepspeech_ios {
+  umbrella header "deepspeech_ios.h"
+
+  export *
+  module * { export * }
```

Comment:
are we sure we're not exposing anything besides our public API here? we have `-fvisibility=hidden`, it'd be bad if we leaked anything

Reply:
This section applies to `deepspeech_ios.h`, not `deepspeech.h`. Even if it did apply to `deepspeech.h`, that header only contains public APIs, so I don't know of any way for a private symbol to leak into this module definition.

Reply:
Right, but I don't know how `modulemap` file is used, it could have been for the linker for example.